mysql-test/suite/versioning/t/vtmd.test

```diff
@@ -197,6 +197,54 @@ set versioning_alter_history= survive;
 create or replace table t (x int) with system versioning;
 select * from t for system_time all;
 
+drop database test;
+create database test;
+use test;
+
+set versioning_alter_history = survive;
+
+create or replace table t (a int) with system versioning;
+insert into t values (1);
+set @ts1 = now(6);
+alter table t add column b int;
+update t set a=2;
+set @ts2 = now(6);
+alter table t drop column b;
+update t set a = 3;
+
+delimiter ~~;
+create or replace procedure select_a_from_range()
+begin
+  select a from t for system_time from '0-0-0' to now(6);
+  select a from t for system_time between '0-0-0' and now(6);
+  select a from t for system_time as of @ts2;
+end~~
+delimiter ;~~
+
+set versioning_identifier_mode = current;
+call select_a_from_range();
+set versioning_identifier_mode = historical_early;
+call select_a_from_range();
+set versioning_identifier_mode = historical;
+call select_a_from_range();
+
+drop database test;
+create database test;
+use test;
+
+set versioning_alter_history = survive;
+set versioning_identifier_mode = current;
```

Comment:
Check
```
set versioning_identifier_mode= default;
select @@versioning_identifier_mode;
```
Please, follow style convention of `= ` (i.e. no space before **=** on assignment).

Reply:
fixed